### ISSUE TITLE
electron-compiler is being resolved related to src directory instead of project directory

### ISSUE THREAD

**Issue by sachetsharma1** [electron-userland/electron-builder#3011]
electron-builder: 20.15.1
electron-compile: 6.1.3

* **Target**: win(nsis)

I am switching from electron-forge to electron-builder to create a nsis package. Following is the configuration: 

  "build": {
    "electronVersion": "2.0.0",
    "win": {
      "target": "NSIS"
    },
    "directories": {
      "buildResources": "./assets/icons",
      "output": "out",
      "app": "./src"
    },
    "publish": null,
    "electronCompile": true
  }

Issue is that electron builder is looking for the electron-compile in the ./src folder not the root folder of the project. 

Error: Cannot find module 'C:\projects\...\src\node_modules\e
lectron-compile\lib\config-parser'

**Comment by popod** [comment]
Yes, this is defined in the [doc](https://www.electron.build/configuration/configuration#MetadataDirectories-app). You should change the `"app": "./src"` property by your root folder and/or correctly set `electronCompile` depending on [this](https://www.electron.build/configuration/configuration#Configuration-electronCompile).

Hope this helps :)

**Comment by sachetsharma1** [comment]
@popod  Not sure what you mean there. electronCompile is just a boolean.
-root 
  -- src 
   -- main.js
   -- node_modules
   -- package.json
 -- node_modules
 -- package.json


build configuration is in the root package.json where I specified "app": "./src". 
builder is looking for electron-compile inside the .src/node_modules. What do I change to resolve it under root/node_modules? Thanks in advance!
